assets/javascript/main.js

```diff
@@ -21,19 +24,44 @@ $(document).ready(function () {
             .then(response => response.json())
             .then(resJSON => console.log(resJSON))
 
-            var results = response.data;
+        var results = response.data;
 
-            for (let i = 0; i < results.length; i++) {
-                const element = array[i];
+        for (let i = 0; i < results.length; i++) {
+            const element = array[i];
 
```

Comment:
Not sure what this for loop is doing, it's using the results and then putting it into a non existent variable named 'array'. This code isn't used, should be removed.

Reply:
I removed it.